src/SelectEventPoll.cc

```diff
@@ -174,6 +174,18 @@ void SelectEventPoll::poll(const struct timeval& tv)
 
   memcpy(&rfds, &rfdset_, sizeof(fd_set));
   memcpy(&wfds, &wfdset_, sizeof(fd_set));
+#ifdef __MINGW32__
+  fd_set efds;
+  FD_ZERO(&efds);
```

Comment:
I'd like to know what happens if `memcpy(&efds, &wfdset_, sizeof(fd_set));` is inserted here.
Does it still cause high cpu usage?